lib/jxl/render_pipeline/stage_write.cc

```diff
@@ -85,7 +85,9 @@ class WriteToU8Stage : public RenderPipelineStage {
         height_(height),
         rgba_(rgba),
         has_alpha_(has_alpha),
-        alpha_c_(alpha_c) {}
+        alpha_c_(alpha_c) {
+    (void)width_;  // unused
```

Comment:
Should we remove the field instead then?

Reply:
Sure.